.coderabbit.yaml

```
@@ -0,0 +1,65 @@
+    # 4. 테스트 코드
+    - path: "src/test/java/**/*.java"
+      instructions: |
+        이 파일은 테스트 코드입니다.
+        1.  **명확한 테스트 이름**: `@DisplayName`을 사용하거나 테스트 메소드 이름을 통해 테스트의 의도가 명확히 드러나는지 확인해주세요. (e.g., `whenUserExists_thenReturnsUserDto`)
+        2.  **BDD 스타일**: Given-When-Then (또는 Arrange-Act-Assert) 구조를 잘 따르는지 검토해주세요. 이는 테스트의 가독성을 크게 향상시킵니다.
+        3.  **Mocking**: 의존하는 객체들을 `@MockBean`이나 `Mockito`를 사용하여 적절히 Mocking하고 있는지, 불필요한 실제 객체를 생성하여 통합 테스트처럼 동작하지는 않는지 확인해주세요.
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: true # 긴 리뷰 요약을 접어서 깔끔하게 표시
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)